typescript-generator-core/src/main/java/cz/habarta/typescript/generator/emitter/TsEnumModel.java

```diff
@@ -12,6 +12,7 @@
 
     private final EnumKind kind;
     private final List<EnumMemberModel> members;
+    private boolean isConstEnum = true;
```

Comment:
could you please replace this field with `isNonConstEnum` field?
it doesn't look as good as your field but since "const" enums are default I would like to use consistently "nonConst" on all places

second thing is that this field should be final (these classes are meant to be immutable)